examples/wistia/src/components/wistia.js

```diff
@@ -0,0 +1,88 @@
+export default class Wistia {
+
+  constructor() {
+    this.videoBrowserEl = document.querySelector('.videoBrowser');
+    this.errorEl = document.querySelector('.error');
+    this.preloaderEl = document.querySelector('.preloader');
+    this.widgetApi = null;
+    this.axios = require('axios');
+  }
+
+  getVideos(projectId, url) {
+    if (!this.widgetApi) {
+      return;
+    }
+    this.axios.get(url, {
+        params: {
+          project_id: projectId
+        }
+      })
+      .then(response => {
+        this.preloaderEl.style.display = 'none';
+
+        var ejsTemplate = require('ejs!./../templates/video-browser.ejs');
+
+        this.videoBrowserEl.innerHTML = ejsTemplate({'title': 'Wistia Widget', 'data': response.data});
+
+        this.inputEl = document.getElementById('wistia-url');
+
+        // Pass value from Contentful to input element.
+        this.inputEl.value = this.widgetApi.field.getValue();
+
+        // TODO (floelhoeffel): Is this a save assumption to access DOM here?
+        let thumbnailEls = document.querySelectorAll('.thumbnail .btn');
+
+        for (let i = 0; i < thumbnailEls.length; i++) {
+          thumbnailEls[i].addEventListener('click', this.onThumbnailClick.bind(this));
+        }
+      })
+      .catch(response => {
+        this.preloaderEl.style.display = 'none';
+        this.errorEl.style.display = 'block';
+        console.error(response);
+      });
+  }
+
+  /**
+   * Calls the callback every time the value of the field is changed by some external event
+   * (e.g. when multiple editors are working on the same entry).
+   */
+  onValueChanged() {
+
+    if (this.widgetApi && this.inputEl) {
+      console.log(this.widgetApi.field.getValue());
+      this.inputEl.value = this.widgetApi.field.getValue();
```

Comment:
`onValueChanged` receives the new value. You should be able to change the signature to `onValueChanged(val)` and just do `this.inputEl.value = val`.